apps/aehttp/src/aehttp_app.erl

```diff
@@ -164,3 +165,7 @@ get_channel_websockets_acceptors() ->
     aeu_env:config_value([<<"websocket">>, <<"channel">>, <<"acceptors">>],
                          aehttp, [channel, websocket, handlers],
                          ?DEFAULT_CHANNEL_ACCEPTORS).
+
+enable_internal_debug_endpoints() ->
+    aeu_env:config_value([<<"http">>, <<"internal">>, <<"debug_endpoints">>],
+                         aehttp, [internal, debug_endpoints], false).
```

Comment:
The default is duplicated from the schema though minor and debatable so ok.

Reply:
👍